configs/defaults/rd_combiner.toml

```diff
@@ -32,15 +32,10 @@ snps_recal_disc_size = 20
 # if false, use a non-preemptible instance to run the ES export
 spot_instance = false
 
-# choices of jar specs to use for the various steps
-# without this setting we would default to matching the native Hail version in the container
-# currently set to 'a8be268326b2ac168035b629a78465c9d94fc7b9' - 0.2.133 + 400'retry + StreamConstraints patch
-[workflow.jar_spec_revisions]
-annotate_cohort = 'a8be268326b2ac168035b629a78465c9d94fc7b9'
-annotate_dataset = 'a8be268326b2ac168035b629a78465c9d94fc7b9'
-combiner = 'a8be268326b2ac168035b629a78465c9d94fc7b9'
-densify = 'a8be268326b2ac168035b629a78465c9d94fc7b9'
-subset = 'a8be268326b2ac168035b629a78465c9d94fc7b9'
+# without this setting we use the native Hail version in the container
+# not currently in use as the cpg-utils default contains these patches by default, plus extra logging
+# most recently set to 'a8be268326b2ac168035b629a78465c9d94fc7b9' - 0.2.133 + 400'retry + StreamConstraints patch
+#default_jar_spec_revision = 'a8be268326b2ac168035b629a78465c9d94fc7b9'
```

Comment:
I'd recommend not putting in a config item called `default_jar_spec_revision`, as that'd be confusing. Nothing will be looking for it in `workflow.es_index` anyway, so uncommenting it would also confusingly do nothing.

Reply:
Glad to see the per-module overrides being removed BTW, now that they're no longer necessary as the default one exists now. But with this being taken care of in cpg-utils and cpg-infrastructure, you shouldn't need any comment here at all really.

Reply:
This was incorrectly nested inside `es_index`, and has been moved to the correct location